library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +35,22 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    public fun Context.createShortcut() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            getSystemService(ShortcutManager::class.java)?.let {
+                val shortcut = ShortcutInfo.Builder(this, "id1")
```

Comment:
Before creating and adding a shortcut you should first check if it does not already exist.

Reply:
Have not found api in `ShortcutManager` to check if it exist, if I should read and write a SP value to check?

Reply:
You can get all shortcuts, map to IDs and check if our ID is in there.